packages/integration-tests/playwright.config.ts

```diff
@@ -2,6 +2,9 @@ import type { PlaywrightTestConfig } from '@playwright/test';
 
 const config: PlaywrightTestConfig = {
   retries: 0,
-  workers: 3,
+  // Run tests inside of a single file in parallel
+  fullyParallel: true,
+  // Use 3 workers on CI, else use defaults (based on available CPU cores)
+  workers: process.env.CI ? 5 : undefined,
```

Comment:
l: Any reason for 5 specifically?

Reply:
No, it was 3 previously and I randomly set it to 5 to see if it still works as expected, and it did (and was _slightly_ faster on CI). FYI without this on CI it defaults to just one worker, because CI has two cores, and the default heuristic is `# of CPUS/2`.
I can also leave it at 3 for CI (but I guess that is equally random 😅 )

Reply:
No 5 is totally fine, I was just wondering 😅

Reply:
I will adjust the comment to make it clear this is an arbitrary number! 😂